packages/perseus/src/widgets/interactive-graph.tsx

```diff
@@ -2351,7 +2352,8 @@ class InteractiveGraph extends React.Component<Props, State> {
             throw makeInvalidTypeError("getAngleEquationString", "angle");
         }
         const coords = InteractiveGraph.getAngleCoords(props.graph, props);
-        const angle = GraphUtils.findAngle(coords[2], coords[0], coords[1]);
+        const allowReflexAngles = props.graph.allowReflexAngles || false;
```

Comment:
Question: Do you still need this after updating the default to false from Anakaren's suggestion?